runtime_builders/builder_util.py

```diff
@@ -36,11 +36,13 @@ def write_to_gcs(gcs_path, file_contents):
         os.write(fd, file_contents)
 
         command = ['gsutil', 'cp', f_name, gcs_path]
+        output = ''
         try:
             output = subprocess.check_output(command)
         except subprocess.CalledProcessError as e:
-            logging.error('Error encountered when writing to GCS!: {0}'
-                          .format(output))
+            logging.error('Error encountered when writing to GCS!')
+            if output is not '':
```

Comment:
hmm yeah not sure what I was thinking here, the output that I care about should actually go out to stdout here I think (based on a test I just did). going to remove this regardless